Tests/SwiftDocCUtilitiesTests/OutOfProcessReferenceResolverTests.swift

```diff
@@ -703,4 +705,153 @@ class OutOfProcessReferenceResolverTests: XCTestCase {
+    ) throws {
+        let testMetadata = OutOfProcessReferenceResolver.ResolvedInformation(
+            kind: .init(name: "Kind Name", id: "com.test.kind.id", isSymbol: true),
+            url: URL(string: "doc://com.test.bundle/something")!,
+            title: "Resolved Title",
+            abstract: "Resolved abstract for this topic.",
+            language: .swift, // This is Swift to account for what is considered a symbol's "first" variant value (rdar://86580516)
+            availableLanguages: [],
+            platforms: platforms,
+            declarationFragments: nil,
+            topicImages: nil,
+            references: nil,
+            variants: []
+        )
+                
+        let resolver = try makeResolver(testMetadata)
+        XCTAssertEqual(resolver.bundleIdentifier, "com.test.bundle", file: file, line: line)
+
+        // Resolve the reference
+        let unresolved = TopicReference.unresolved(
+            UnresolvedTopicReference(topicURL: ValidatedURL(parsingExact: "doc://com.test.bundle/something")!))
+        guard case .success(let resolvedReference) = resolver.resolve(unresolved) else {
+            XCTFail("Unexpectedly failed to resolve reference")
+            return
+        }
+        
+        // Resolve the symbol
+        let topicLinkEntity = resolver.entity(with: resolvedReference)
+
+        XCTAssertEqual(topicLinkEntity.topicRenderReference.isBeta, isBeta, file: file, line: line)
+        
+        // Resolve the symbol
+        let (_, symbolEntity) = try XCTUnwrap(resolver.symbolReferenceAndEntity(withPreciseIdentifier: "abc123"), "Unexpectedly failed to resolve symbol")
+        
+        XCTAssertEqual(symbolEntity.topicRenderReference.isBeta, isBeta, file: file, line: line)
+
+    }
+    
+    func testResolvingSymbolBetaStatusProcess() throws {
+        #if os(macOS)
+        func makeResolver(testMetadata: OutOfProcessReferenceResolver.ResolvedInformation) throws -> OutOfProcessReferenceResolver {
+            let temporaryFolder = try createTemporaryDirectory()
+            let executableLocation = temporaryFolder.appendingPathComponent("link-resolver-executable")
+            
+            let encodedMetadata = try String(data: JSONEncoder().encode(testMetadata), encoding: .utf8)!
+            
+            try """
+        #!/bin/bash
+        echo '{"bundleIdentifier":"com.test.bundle"}'            # Write this resolver's bundle identifier
+        read                                                     # Wait for docc to send a symbol USR
+        echo '{"resolvedInformation":\(encodedMetadata)}'     # Respond with the test metadata (above)
+        read                                                     # Wait for docc to send a symbol USR
+        echo '{"resolvedInformation":\(encodedMetadata)}'     # Respond with the test metadata (above)
+        """.write(to: executableLocation, atomically: true, encoding: .utf8)
```

Comment:
nit: This bash script string has comments on each line but two of the lines are not aligned with the rest.

Reply:
Fixed :)